### ISSUE TITLE
The Contrast between "Resources I haven't viewed" and "Resources I've viewed" is minimal

### ISSUE THREAD

**Issue by zqian** [tl-its-umich-edu/my-learning-analytics#798]
## Expected behavior (A clear and concise description of what you expected to happen) : 

Contrast betweem "Resources I haven't viewed" and "Resources I've viewed" should be increased to assist with Color Impaired users

## Describe the bug (Tell us what happens instead of the expected behavior) :

Color impaired users will not be able to tell at a glance the difference between "Resources I haven't viewed" and "Resources I've viewed" bars in the visualization.

## Steps to Reproduce :
1. Download a proper Color contrast checker for the browser
2. Change the colors to a setting that works well when switching view to grayscale.

**Comment by ssciolla** [comment]
This is being addressed by PR #775 and is related to issue #748.

**Comment by jennlove-um** [comment]
This was resolved by #748

**Comment by gsilver** [comment]
@jennlove-um @pushyamig - looking good.  Difference between the two is distinguishable after 50% contrast loss and total achromatopsia.  Mark as done.
